### DISCUSSION TITLE
table组件是否支持控制显示所需要的列功能？

### DISCUSSION THREAD

**Discussion by githubqh** [ant-design/ant-design#36109] [Q&A]
就是想看哪些列的数据就选择这些列，不想看的就隐藏掉这些列，是否有这个功能？

**Comment by huntdream** [comment]
自己动态筛一下`columns`就行
可以参考：https://procomponents.ant.design/components/table?current=1&pageSize=5
<img width="802" alt="image" src="https://user-images.githubusercontent.com/7903755/174522244-d31704ff-6a13-4bc3-8ee4-f4ed50d728d6.png">
